<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExposureForm</class>
 <widget class="QWidget" name="ExposureForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>125</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="MainLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="ptChoice" name="ExposureMode"/>
      </item>
      <item>
       <widget class="ptChoice" name="ClipMode"/>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>5</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ptInput" name="Exposure"/>
   </item>
   <item>
    <widget class="QWidget" name="AutoCfgWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Automatic exposure calculation</string>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ptInput" name="WhiteFraction"/>
      </item>
      <item>
       <widget class="ptInput" name="WhiteLevel"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ptInput</class>
   <extends>QProgressBar</extends>
   <header>ptInput.h</header>
  </customwidget>
  <customwidget>
   <class>ptChoice</class>
   <extends>QComboBox</extends>
   <header>ptChoice.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
